### PR TITLE
Updated Russian translation

### PR DESCRIPTION
**Pull request by ghost**
Translating iceweasel strings into Russian

### CODE REVIEW

File: app/src/main/res/values-ru/static_strings.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Name of the application -->
+    <string name="app_name" translatable="false">Iceweasel Preview</string>
+
+    <!-- Preference for developers -->
+    <string name="preference_leakcanary" translatable="false">LeakCanary</string>
```
Comment (abhijitvalluri) [inline app/src/main/res/values-ru/static_strings.xml]: This looks good. However, I see a lot of translations for strings that are specifically marked as `translatable="false"`. This flag means that they should not have translations in any other file and so these entries should be removed from this file as well.

**Review by abhijitvalluri — CHANGES_REQUESTED** [review]
This looks good. But please remove translations for strings that should not be translated.

**Comment by NikGreens** [comment]
Done

**Comment by ghost** [comment]
Done, I fixed it

**Review by abhijitvalluri — APPROVED** [review]
Thanks!

**Pull request merged**